packages/core/src/UserState.ts

```diff
@@ -9,26 +10,29 @@ import {
     ZkIdentity,
 } from '@unirep/crypto'
 import { Circuit, genProofAndPublicSignals } from '@unirep/circuits'
+import { IAttestation, Attestation } from '@unirep/contracts'
+import { NUM_ATTESTATIONS_PER_PROOF } from '@unirep/config'
+
 import {
     defaultUserStateLeaf,
     genEpochKey,
     genNewSMT,
     genEpochKeyNullifier,
     genReputationNullifier,
+    stringifyAttestation,
```

Comment:
`IAttestation` defined in @unirep/contracts. I think this more sense if we move `stringifyAttestation` to `@unirep/contracts`. How you think about it?

Reply:
I think only `core` requires stringify function so that I put them in `core`.
Or it can be extended from `@unirep/contracts`